docs/example-scenario/data/data-warehouse.md

```diff
@@ -71,28 +71,28 @@ Data is loaded from these different data sources using several Azure components:
 The technologies in this architecture were chosen because they met the company's requirements for scalability and availability, while helping them control costs.
 
 * The [massively parallel processing architecture](/azure/sql-data-warehouse/massively-parallel-processing-mpp-architecture) of SQL Data Warehouse provides scalability and high performance.
-* SQL Data Warehouse has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/sql-data-warehouse/) and [recommended practices for achieving high availability](/azure/sql-data-warehouse/sql-data-warehouse-best-practices).
+* SQL Data Warehouse has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/sql-data-warehouse) and [recommended practices for achieving high availability](/azure/sql-data-warehouse/sql-data-warehouse-best-practices).
 * When analysis activity is low, the company can [scale SQL Data Warehouse on demand](/azure/sql-data-warehouse/sql-data-warehouse-manage-compute-overview), reducing or even pausing compute to lower costs.
 * Azure Analysis Services can be [scaled out](/azure/analysis-services/analysis-services-scale-out) to reduce response times during high query workloads. You can also separate processing from the query pool, so that client queries aren't slowed down by processing operations. 
-* Azure Analysis Services also has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/analysis-services/v1_0/) and [recommended practices for achieving high availability](/azure/analysis-services/analysis-services-bcdr).
+* Azure Analysis Services also has [guaranteed SLAs](https://azure.microsoft.com/support/legal/sla/analysis-services/v1_0) and [recommended practices for achieving high availability](/azure/analysis-services/analysis-services-bcdr).
```

Comment:
Use this link for the SLA: `https://azure.microsoft.com/support/legal/sla/analysis-services` 

- leave out the version number, it will redirect appropriately